experimental/GModule/GModule.jl

```diff
@@ -789,7 +789,7 @@ function Oscar.composition_factors_with_multiplicity(C::GModule{<:Any, <:Generic
 end
 
 """
-A decomposition of the module into indecomposable summands. Returns a list
+Return a decomposition of the module `C` into indecomposable summands as a list
```

Comment:
I don't think that mentioning `C` is necessary here, as the docstring does not specify names for the input.

Reply:
Of course most docstrings in that file are just rough sketches to start with -- the plan still is to clean them all up and migrate them to the "regular" codebase... (CC @ThomasBreuer)